docs/manual/extend.md

```diff
@@ -11,6 +11,15 @@ find_package(remage REQUIRED)
 target_link_libraries(myapp PRIVATE RMG::remage)
 ```
 
+:::{note}
+
+When updating from older _remage_ versions you might encounter some issues with
+remnants of old dependencies. In this case fully delete your current _remage_
+installation and re-install into a clean folder, as the install process itself
+will not remove any files that are not on the list of to-be-installed files.
+
+:::
```

Comment:
I find hard to understand what this means, I would simplify by just saying that if there's any unexpected error one should always try to compile from scratch (i.e. remove the build folder). also, I would move this in install.md